### DISCUSSION TITLE
Has someone written a Premake file?

### DISCUSSION THREAD

**Discussion by asu4net** [raysan5/raylib#4099] [Q&A] [answered]
Hi, it would be nice to include raylib in my engine in order to make the graphic stuff more simple. Currently I'm using premake to link the third party libraries. I know I could use cmake and then, inspect the project settings in order to writte my own premake file, but if someone has already done the job it would be appreciated :3

**Comment by raysan5** [comment, marked as answer]
@asu4net Afaik, @JeffM2501 uses premake.

**Reply by JeffM2501** [reply]
game-premake is a system to setup build systems with raylib using premake
https://drive.google.com/drive/folders/1o7W61F_7j7tSEpa5uEKhlVVYZqPklO_a?usp=drive_link
